manifests/init.pp

```diff
@@ -52,6 +54,7 @@
 #
 class foreman_proxy_content (
   String[1] $parent_fqdn = $foreman_proxy_content::params::parent_fqdn,
+  String[1] $foreman_fqdn = $foreman_proxy_content::params::foreman_fqdn,
```

Comment:
I think this could be part of a later change if we figure it out :) this is just an initial step to clean up this module and get pulp master setup into this